### ISSUE TITLE
Wrong language detection

### ISSUE THREAD

**Issue by toxicrecker** [github/linguist#5393]
Me and my friend are making a game using [Godot Engine](https://godotengine.org) and are using github as the platform to share code, however in our repository github is detecting GDScript as GAP for some reason which messes up the syntax highlighting. Is there a way to fix this?

**Comment by Alhadis** [comment]
> Is there a way to fix this?

Yes, there was a link to the solution in the issue template which you didn't bother reading/filling out.

**Comment by toxicrecker** [comment]
> > Is there a way to fix this?
> 
> Yes, there was a link to the solution in the issue template which you didn't bother reading/filling out.

Sorry but I can't find any links in the template

**Comment by Alhadis** [comment]
Sorry, I forgot we removed the URLs after moving to [Discussions](https://github.com/github/linguist/discussions) for these sorts of issues.

Which brings me neatly along to another point — you missed *three* clear instructions to check the Discussions tab first instead of opening an issue for this:

> <img src="https://user-images.githubusercontent.com/2346707/119985432-f86a0800-c005-11eb-99e6-c817888ec8c4.png" width="880" alt="Figure 1" />
>
> <img src="https://user-images.githubusercontent.com/2346707/119985449-fc962580-c005-11eb-826e-8e5e88eb3f1d.png" width="914" alt="Figure 2" />

This would have led you to our [documentation on overrides](https://github.com/github/linguist/blob/master/docs/overrides.md), which contains the solution for your problem.
